src/test/java/io/cryostat/net/reports/ActiveRecordingReportCacheTest.java

```diff
@@ -86,21 +84,21 @@
     class TestSubprocessReportGenerator extends SubprocessReportGenerator {
```

Comment:
Where is this nested class used? It seems the only place it's referenced is in the class and constructor definition.

Reply:
Ah right, nice catch. I think it used to actually be used in tests but it was replaced by a more typical mock at some point. I'll take it out.